### PR TITLE
IGAPP-723: Hide news tabs

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

http://0.0.0.0:9000/testumgebung/de/news/tu-news

### CODE REVIEW

File: native/src/components/NewsListItem.tsx [hunk 1]
```diff
@@ -41,11 +43,11 @@ const ListItemWrapper = styled.View`
 const StyledTouchableOpacity = styled.TouchableOpacity`
   flex-direction: column;
 `
-const Divider = styled.View`
+const Divider = styled(View)<{ firstItem?: boolean }>`
```
Comment (klinzo) [inline native/src/components/NewsListItem.tsx:46]: Why did you change `.View` to `(View)`?

Comment (klinzo) [reply native/src/components/NewsListItem.tsx:46]:
```suggestion
const Divider = styled(View)<{ firstItem: boolean }>`
```

Comment (f1sh1918) [reply native/src/components/NewsListItem.tsx:46]: hm firstItem should be optional for the divider!?

Comment (klinzo) [reply native/src/components/NewsListItem.tsx:46]: Sounds good :)

**Review by klinzo — COMMENTED** [review]
I am getting, seems to be new:
> Warning: Each child in a list should have a unique "key" prop.

Tested on firefox and iOS, I really like it :+1:

**Review by klinzo — APPROVED** [review]

**Review by Taggotty — APPROVED** [review]
